client/components/navbar.jsx

```diff
@@ -1,17 +1,71 @@
 import React from 'react';
 import 'bootstrap-icons/font/bootstrap-icons.css';
-export default function NavBar(props) {
-  const style = {
-    fontSize: '2rem',
-    color: 'white',
-    paddingLeft: '2rem'
-  };
-  return (
-      <div className='row'>
-        <div className='nav-bar'>
-        <i className="bi bi-justify bi--2xl" style={style}></i>
+import Container from 'react-bootstrap/Container';
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Delete this import, the Bootstrap CSS only needs to be imported in one place in the App component.